integration_tests/helpers/envHelper.ts

```diff
@@ -0,0 +1,5 @@
+function resourcePrefix() {
+  return `di-btm${process.env.ENV_NAME ? '-' + process.env.ENV_NAME : ''}`;
```

Comment:
Shouldn't the `di-btm` prefix part of this be pulled in from config as well?

Reply:
There is currently nowhere I could pull it from. If we were to create another project to deploy separately, it would probably have it's own set of tests, so this would be hard-coded as something else there.